### PR TITLE
Evaluate nested floor/ceiling.

### LINKED ISSUE #18689: Nested floor and ceiling don't fully evaluate
```
>>> floor(floor(x) + 1) + x
x + floor(floor(x)) + 1
>>> x + floor(floor(x)) + 1
x + floor(x) + 1
```

Same idea with `ceiling`.

### PR DESCRIPTION
**Pull request by sbt4104**
#### References to other Issues or PRs
Fixes #18689 

#### Brief description of what is fixed or changed
evaluate nested floor and ceiling.
example-
```python
floor(floor(floor(x)) + 3) == floor(x) + 3
ceiling(ceiling(ceiling(x)) + 3) == ceiling(x) + 3
ceiling(ceiling(floor(x)) + 3) == floor(x) + 3
```
### Release Notes

* functions
    - Evaluate nested floor/ceiling.

### CODE REVIEW

File: sympy/functions/elementary/integers.py [hunk 1]
```diff
@@ -68,6 +68,8 @@ def eval(cls, arg):
             return ipart
         elif spart.is_imaginary or (S.ImaginaryUnit*spart).is_real:
             return ipart + cls(im(spart), evaluate=False)*S.ImaginaryUnit
+        elif isinstance(spart,cls):
```
Comment (Smit-create) [inline sympy/functions/elementary/integers.py]: Please add spacing around commas.
`elif isinstance(spart, cls):`

**Comment by Smit-create** [comment]
Please add test cases for the change you have made

File: sympy/functions/elementary/integers.py (hunk 1 above)
Comment (sbt4104) [reply sympy/functions/elementary/integers.py]: Please check the changes.

**Comment by smichr** [comment]
Looks good.

**Pull request merged**